test/com/facebook/buck/python/PythonBuckConfigTest.java

```diff
@@ -266,6 +266,32 @@ public void testGetPyrunVersion() throws Exception {
   }
 
   @Test
+  public void testGetPypyVersion() throws Exception {
+    String pypyOutput =
+        "Python 2.7.10 (850edf14b2c75573720f59e95767335fb1affe55, Oct 30 2015, 00:18:28)\n" +
+        "[PyPy 4.0.0 with GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.1.76)]\n";
+
+    PythonVersion version =
+        PythonBuckConfig.extractPythonVersion(
+            Paths.get("non", "important", "path"),
+            new ProcessExecutor.Result(0, "", pypyOutput));
+    assertEquals("Python 2.7", version.toString());
```

Comment:
We generally prefer `assertThat` with a `Matcher` as the test below the two new ones you've added.